src/smc-webapp/printing.coffee

```diff
@@ -547,6 +557,36 @@ class SagewsPrinter extends Printer
                 cb        : (err) ->
                     cb(err, data_base64)
 
+        $html = null
```

Comment:
For the record, I think we've never once used a variable name starting with $ in all the SMC source code, since it's not valid in Python, and I try to have a somewhat pythonic style.